wintasee/hooks/openglhooks.cpp

```diff
@@ -769,11 +772,11 @@ struct OpenGLDisplayListEntry
 		args.clear();
 		if(buffer)
 		{
-#if _MSC_VER > 1310
-			_aligned_free(buffer);
-#else
-			free(buffer);
-#endif
+//#if _MSC_VER > 1310
+//			_aligned_free(buffer);
+//#else
+            MemoryManager::Deallocate(buffer);
+//#endif

```

Comment:
These I want to keep, just in case the _aligned_ functions do something I have yet to understand.